plugins/clipboard.client.js

```diff
@@ -0,0 +1,30 @@
+import { useNotifications } from '~/plugins/notifications'
+
+export default ({ $t }, inject) => {
+  const { add: addNotification } = useNotifications()
+
+  inject('clipboard', {
+    copy: (text, successMsg, failureMsg) => {
+      if (!navigator.clipboard) {
```

Comment:
Maybe a simple `location.replace` if not?

Reply:
Thanks for the review. But I don't get it 😅 
What do you mean? There should not be any redirect 🤔

Reply:
We can return `false` here or throw an unsupported error and handle it in `copyEmailToClipboard`. My point is, we are preventing the default behavior of link clicking and changing it to "copy to clipboard". While this can be solved by doing a programmatic navigation that bots cannot follow (same as [email obfuscation in cloudflare](https://support.cloudflare.com/hc/en-us/articles/200170016-What-is-Email-Address-Obfuscation-))

Reply:
Oh yes, I see what you mean. Agreed, we should still do something even if Clipboard is not supported.
I like the simple plan of returning false and then let the `mailto` proceed. Updating it!